test/java/src/main/resources/test.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
please rename to `test_1.json` for consistency (https://github.com/juliaogris/sysl/tree/json_serializer_mv/test/java/resources)